src/core/blueprint/into_schema.rs

```diff
@@ -42,17 +43,37 @@ fn to_type(def: &Definition) -> dynamic::Type {
                 let field = field.clone();
                 let type_ref = to_type_ref(&field.of_type);
                 let field_name = &field.name.clone();
+                let args = &field.args;
+                let args = args
+                    .iter()
+                    .filter_map(|v| {
+                        let val = v.default_value.clone()?;
+                        if val.is_null() {
+                            return None;
+                        }
+                        Some((v.name.clone(), val))
+                    })
+                    .collect::<HashMap<String, serde_json::Value>>();
+                let args = if args.is_empty() {
+                    ConstValue::Null
+                } else {
+                    async_graphql_value::to_value(args).unwrap_or_default()
+                };
+
                 let mut dyn_schema_field = dynamic::Field::new(
```

Comment:
Just call the the `dyn_schema_field.arguments(args)` Current implementation is a bit complicated and unnecessary.